po/meson.build

```
@@ -2,3 +2,4 @@ i18n.gettext(gettext_name,
     args: '--directory=' + meson.source_root(),
     preset: 'glib'
 )
+subdir('extra')
```

```suggestion

subdir('extra')
```

This is a tiny thing, but we may want to add a new line before this line to follow conventions in other repositories.